### ISSUE TITLE
Doesn't work during debugging.

### ISSUE THREAD

**Issue by Blizzke** [albertz/IdeaResolveSymlinks#2]
I have a python project where the context directory is set to the symlinked version.
During normal development this works excellent, only downside is that when it changes the path to the physical path instead of the symlinked one, I get the "not part of your project"-popup, but that is my choice.

Unfortunately during debugging things go haywire. The IDE wants to open the symlinked version to debug, your plugin closes it and opens the physical one. The IDE then sees that the requested file is not open (i presume) and tries to open it again, plugin closes it again etc.
This is just an eternal fight (even without touching anything) that makes it unusable for me (unfortunately, because its exactly what I'm looking for).

**Comment by albertz** [comment]
Yep. I stumbled upon this problem as well recently. I didn't had time yet to look into it.

But also, I currently don't have a good idea how to tackle this problem. Any suggestions or contributions are more than welcome.

I also don't really have experience in IDEA plugins at all, so I don't have a good understanding of what is possible. I assume if you could hook into the debugging part which opens the file, and change that somehow, this could be a way to solve this. Not sure if this is possible.

**Comment by albertz** [comment]
As a workaround, to at least make it usable for debugging, is to ignore any open request when the user is debugging. I assume this would be possible. I don't know how to check the current state, if we are debugging, or the file was requested to be opened by the debugger, but this is probably possible somehow.

**Comment by albertz** [comment]
Btw, w.r.t. the message of the "not part of your project" popup, I just found [Non Project Files Unlocker
](https://plugins.jetbrains.com/plugin/7635-non-project-files-unlocker) ([GitHub](https://github.com/krasa/NonProjectFilesUnlocker)).

**Comment by albertz** [comment]
Maybe I have to use `source.updateFilePresentation()` instead or so?

**Comment by Blizzke** [comment]
> Maybe I have to use `source.updateFilePresentation()` instead or so?

Sorry I have no idea, no experience with intellij plugins whatsoever. 
Perhaps you could ask jetbrains themselves? 

As for the "ignoring the open dialog", that's not really possible in my case. When I start a debug session for that specific project the entry file itself is symlinked, so it will be opened right away.  The dialog does not appear because it's a local file.

**Comment by albertz** [comment]
I implemented some hacky workaround for this issue. Not sure if this is a good solution. But it works for me at least...

I made a new release v1.2 for this.

**Comment by albertz** [comment]
I still see that some people seem to have problems with this.
